contracts/toCKB-lockscript/src/main.rs

```diff
@@ -57,16 +50,15 @@ impl From<SysError> for Error {
 }
 
 fn main() -> Result<(), Error> {
-    // remove below examples and write your code here
-
-    let script = load_script()?;
-    let args: Bytes = script.args().unpack();
-    debug!("script args is {:?}", args);
-
-    let tx_hash = load_tx_hash()?;
-    debug!("tx hash is {:?}", tx_hash);
+    verify()
+}
 
-    let _buf: Vec<_> = vec![0u8; 32];
+fn verify() -> Result<(), Error> {
+    let args: Bytes = load_script()?.args().unpack();
+    let count = QueryIter::new(load_cell_type, Source::GroupInput)
+        .filter(|type_script_opt| (type_script_opt.clone().unwrap().as_slice()[0..54] != args[..]))
```

Comment:
1. 
```suggestion
        .filter(|type_script_opt| type_script_opt.is_some())
        .filter(|type_script_opt| (type_script_opt.clone().unwrap().as_slice()[0..54] != args[..]))
```

2. the logic is wrong
should verify count == 0

Reply:
done